core/src/test/scala/neotypes/BaseIntegrationSpec.scala

```diff
@@ -21,53 +20,55 @@ abstract class BaseIntegrationSpec[F[_]](testkit: EffectTestkit[F]) extends Base
       .configure(_.addEnv("NEO4JLABS_PLUGINS", "[\"graph-data-science\"]"))
       .configure(_.withImagePullPolicy(PullPolicy.ageBased(toJavaDuration(1.day))))
 
-  protected lazy final val driver =
+  protected final lazy val driver =
     neo4j.GraphDatabase.driver(container.boltUrl)
 
-  private lazy final val neo4jSession =
-    driver.session()
```

Comment:
Keeping two sessions was causing troubles, so I had to remove it.
Anyways, this change allows us to switch between the async and the streaming session so we can ensure the new wrappers are totally backwards compatible.